web/src/pages/index.tsx

```diff
@@ -75,7 +75,7 @@ const ShowCitation: React.FC<{citation: Citation, i: number}> = ({citation, i})
 
     // if we don't have a url, link to a duckduckgo search for the title instead
     const url = citation.url && citation.url !== ""
-                ? citation.url 
+                ? citation.url
                 : `https://duckduckgo.com/?q=${encodeURIComponent(citation.title)}`;
```

Comment:
Nice! I didn't notice the duckduckgo search before. Hopefully won't be needed as much after ARD is cleaned up. One thing we discussed that would be nice (eventually) is to check if the citation.url is valid (returns content without errors) and give an alternate result if the it's down or removed for some reason

Reply:
Absolutely. It's really meant as more of a stopgap, but should be nice as a fallback if stuff still slips through the cracks.

On the topic, at some point we might want to have some system in place for flagging "removed" sources, since them being "down" can mean anything from a literal 404, or temporarily down site page, or non-response, to just an internal redirect, or private page. Not really any good automated way to detect the later camp.